omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+``WriteScriptRepo``, ``Chgrp``. A facility manager importing on behalf
+of other users may appropriately be given all but the ``Sudo``
+restriction. A human resources representative may be given all but the
+``ModifyGroup``, ``ModifyGroupMembership``, ``ModifyUser`` restrictions.
+Much depends on how personnel roles are handled in each specific
+institution.
+
+.. warning::
+
+    The ``ReadSession`` restriction should be applied to *all*
+    restricted administrators. Without that restriction members of the
+    ``system`` group can read other users' session UUIDs and join those
+    sessions as the user. In contrast, while an administrator without
+    the ``Sudo`` restriction can become other users, the security system
+    prevents their using sudo to elevate their own administrative
+    powers. An administrator cannot sudo to become ``root`` to escape
+    their restrictions. :javadoc:`A security filter
+    <ome/security/basic/LightAdminPrivilegesSecurityFilter.html>`
+    assists in enforcing ``ReadSession``.
+
+
+Working with restrictions
+-------------------------
+
+Restricted administrators
+"""""""""""""""""""""""""
+
+The :javadoc:`admin service <slice2html/omero/api/IAdmin.html>` offers
+new operations for managing restrictions on administrators.
+``createLightSystemUser`` creates a new restricted administrator.
+``getAdminPrivileges`` and ``setAdminPrivileges`` manage the
+restrictions on an existing administrator.
+
+Using ``setAdminPrivileges`` to set an empty list of privileges fills
+that user's ``Experimenter.config`` property with a ``false`` value for
+every ``AdminPrivilege`` name. That user does not thus become like a
+normal user: they retain all administrative powers not explicitly
+restricted, such as being able to read all users' images. A normal
+user's restrictions can be set: they simply have no effect if the user
+is not an administrator as there is then nothing to restrict.
+
+``getCurrentAdminPrivileges`` is useful for OMERO clients to find how
+the currently logged-in administrator is restricted.
+``getAdminsWithPrivileges`` identifies the administrators who are
+sufficiently unrestricted in a given way so that users know which
+administrators may be able to help them.
+
+
+Permissions on model objects
+""""""""""""""""""""""""""""
+
+OMERO model objects have a :ref:`details property <model details
+property>` that bears information on :ref:`object permissions
+<permissions object>`; the ``canChgrp`` and ``canChown`` methods were
+introduced in OMERO 5.4. Client software may find these permissions
+methods a useful guide as to what the current administrator may do to
+which objects.
+
+
+Event context
+"""""""""""""
+
+Since OMERO 5.4 the :javadoc:`event context
+<slice2html/omero/sys/EventContext.html>` for the current session,
+available from the admin service, has additional data members:
+
+* ``adminPrivileges`` that lists the restrictions *not* applying to the
```

Comment:
``restrictions`` and ``not applying`` is a double negative which makes the sentence hard to read. As the name of the ``member`` suggests, these are privileges. Maybe use one more sentence using the word ``privileges`` to ease the reading ?